h/models/group_scope.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import sqlalchemy as sa
+
+from h.db import Base
+
+
+class GroupScope(Base):
+    __tablename__ = 'groupscope'
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    #: A fully qualified domain name, e.g. example.com, www.nytimes.com or
```

Comment:
Technically, not yet, right?

Reply:
I guess not. On the other hand this is what it's _intended_ to be, and I'm not yet sure where any validation will go when we have it (model? service? view?). So I'm not sure whether I want to remove this docstring or not